procedures/foreman/branch.md.erb

```diff
@@ -12,98 +14,114 @@
+  - [ ] Create [releases/<%= release %>/RPM-GPG-KEY-foreman](https://yum.theforeman.org/releases/<%= release %>/RPM-GPG-KEY-foreman) on yum.theforeman.org
   - [ ] Commit the [settings file](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/foreman/<%= release %>/settings) to the `theforeman-rel-eng` repository
 - [ ] Create new settings files for [client](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/client/<%= release %>/settings), ensure it has the rights `OSES` list and commit it too.
 
-# Package build systems
-
-- [ ] Create tool_belt config based on the previous release and the nightly config which might contain different tags/repos.
-- [ ] Clone tags and create build targets in Koji: `bundle exec ./tools.rb koji --commit configs/foreman/<%= release %>.yaml`
-- [ ] Make sure all nightly builds of packages are removed from the releases tags
-- [ ] Create mash configuration on Koji:
-  ```
-  bundle exec ./tools.rb mash-scripts configs/foreman/<%= release %>.yaml
-  sed -i '/strict_keys/ s/True/False/' mash_scripts/foreman/<%= release %>.0/foreman-plugins-<%= release %>-*.mash
-  scp mash_scripts/foreman/<%= release %>.0/*.mash root@koji.katello.org:/etc/mash/
-  ```
-- [ ] Update [mash scripts](https://github.com/theforeman/foreman-infra/tree/master/koji/) and deploy them to Koji:
-  - [ ] Verify main mash scripts (`foreman-mash-split.py`, `foreman-rails-mash-split.py` and `foreman-plugin-mash-split.py`) don't need any version-specific updates
-  - [ ] Update dists in client mash script (`foreman-client-mash-split.py`)
-- [ ] Add version <%= release %> to jobs in axes and/or combination filters, remove old ones (keep three)
-- [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Branching-for-release)
-
-# Branch main code repos
-
-- Create <%= release %>-stable branches:
+# Branching
+
+## Release Engineer
+
+- [ ] Branch RPM packaging using [tool_belt](https://github.com/theforeman/tool_belt)
+  - [ ] Create `configs/foreman/<%= release %>.yaml` based on the nightly and previous release
+  - [ ] Clone tags and create build targets in Koji: `bundle exec ./tools.rb koji --commit configs/foreman/<%= release %>.yaml`
+  - [ ] Create `rpm/<%= release %>` in [foreman-packaging](https://github.com/theforeman/foreman-packaging) based on `rpm/develop`
+- [ ] Branch Debian packaging
+  - [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Branching-for-release)
+  - [ ] Create `deb/<%= release %>` in [foreman-packaging](https://github.com/theforeman/foreman-packaging) based on `deb/develop`
+
+## Release Owner
+
+- [ ] Create <%= release %>-stable branches
```

Comment:
As mentioned in the last release meeting, I think this is also where we should branch hammer.